configs/_base_/datasets/kittistep.py

```diff
@@ -0,0 +1,54 @@
+# dataset settings
+dataset_type = 'KITTISTEPDataset'
+data_root = 'data/kitti_step/'
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+crop_size = (368, 368)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations'),
+    dict(type='Resize', img_scale=(1242, 375), ratio_range=(0.5, 2.0)),
+    dict(type='RandomCrop', crop_size=crop_size, cat_max_ratio=0.75),
+    dict(type='RandomFlip', prob=0.5),
+    dict(type='PhotoMetricDistortion'),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='Pad', size=crop_size, pad_val=0, seg_pad_val=255),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_semantic_seg']),
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='MultiScaleFlipAug',
+        img_scale=(1242, 375),
+        # img_ratios=[0.5, 0.75, 1.0, 1.25, 1.5, 1.75],
+        flip=False,
+        transforms=[
+            dict(type='Resize', keep_ratio=True),
+            dict(type='RandomFlip'),
+            dict(type='Normalize', **img_norm_cfg),
+            dict(type='ImageToTensor', keys=['img']),
+            dict(type='Collect', keys=['img']),
+        ])
+]
+data = dict(
+    samples_per_gpu=2,
+    workers_per_gpu=2,
+    train=dict(
+        type=dataset_type,
+        data_root=data_root,
+        img_dir='training_openmmlab/image_02/train',
+        ann_dir='panoptic_maps_openmmlab/train',
```

Comment:
I have a little question, is KITTI-STEP for panoptic segmentation tasks or semantic segmentation? MMSeg only works on semantic segmentation, and panoptic segmentation and instance segmentation are supported in MMDet

Reply:
KITTI-STEP is for panoptic segmentation. But because it follows the Cityscapes label set and the same camera setting (car dashcam view), it is a beautiful dataset to compare sem-seg methods especially if they address ML-interpretatbility. E.g., with this dataset you an compare interclass dependencies or interclass confusion on similar datasets.

Reply:
I see. Thanks for your explanation.
Is there any public research work using it as a sem-seg dataset, since the baseline result in MMSeg needs a public reference?

Reply:
Thanks for your effort. The closest baseline can be seen here: http://www.cvlibs.net/publications/Weber2021NEURIPSDATA.pdf in the appendix G. They report STQ metric which is based on SQ(=mIoU). So far, there is no published work using it as a pure Sem-seg dataset. But i am working on it so that i hope it will in some months :) If you think it is more convenient, we also could wait some months and I can create a new PR after publication.

Reply:
Of course, we really appreciate it that users release their work in mmseg!

Reply:
So should i wait with the dataset PR until i used it in a publication?